src/wrappers/themis/java/build.gradle

```diff
@@ -25,10 +31,10 @@ archivesBaseName = 'java-themis'
 version = javaThemisVersion
 group = 'com.cossacklabs'
 
-// Compile for Java 8.
+// Compile for Java 7.
```

Comment:
java 7? 🤔

Reply:
> Java 7?

Modern Android systems support most of Java 8 features, but not quite everything, and there may be some older systems that do not support Java 8. Obviously, Java 9 and later are off limits as well (and will probably remain that way in the nearest future). So we’re effectively limited to Java 7 feature set on Android.

Since desktop Java builds from the same source code, it’s a good idea to limit it to Java 7 as well. it’s likely that given Gradle support for desktop Java, most of the development will happen there, so I don’t want to accidentally use Java 8 features and find out later that Android cannot compile that.

Though, it kinda sucks since Java 8 has so many goodies, like lambdas, default implementations in interfaces, etc. Well, we can live with that since Themis is just a simple wrapper.